api/rds.proto

```diff
@@ -198,9 +198,18 @@ message RouteAction {
   RoutingPriority priority = 11;
 
   // Specifies a set of headers that will be added to requests matching this
-  // route.
+  // route. In the presence of duplicate header keys, precedence rules apply.
```

Comment:
What are these precedence rules?

Reply:
Do you mean duplicate headers at each level of the hierarchy? I agree the comment is a bit confusing.

Reply:
I cargo culted it from the existing description of the RouteConfiguration's request_headers_to_add. I'll see if I can come up with something better.